src/checkbox.scss

```diff
@@ -169,3 +170,61 @@ $block: #{$fd-namespace}-checkbox;
     }
   }
 }
+
+.#{$displayonlyblock} {
```

Comment:
I would go more for `.is-readonly / readonly` state instad of having another block class. So class: `fd-checkbox is-readonly`. And on css `&.is-readonly + fd-checkbox__label {}`